### ISSUE TITLE
Add Breaks During Game Sessions

### ISSUE THREAD

**Issue by becurrie** [becurrie/titandash#32]
I do not believe that this is tracked in any sense by GH, but adding some breaks here and there every X hours may be something worthwhile to add, it will be optional of course, and can just follow the normal configurations options present now.

```
[ACTIONS_BREAKING]
# Enable breaks in game.
ENABLE_BREAKS = off
BREAK_PLUS_MINUS = 20
# Determine the amount of minutes that must be played consecutively (plus/minus a random amount between the BREAK_PLUS_MINUS value.
BREAK_MINUTES_REQUIRED = 90
# Specify the maximum amount of time that a break can take place.
BREAK_MINUTES_MAX = 20
# Specify the minimum amount of time that a break can take place.
BREAK_MINUTES_MIN = 5
```

This should do for now.

**Comment by becurrie** [comment]
Reopening this since it seems like a useful addition to curb the time since install and time played to look a bit more reasonable if the user wishes.

**Comment by becurrie** [comment]
Resolved by #129
